README.md

```diff
@@ -82,6 +82,7 @@ Plugin supports special global variables which are allow to change behaviour of
  - **`g:cmake_build_executor`** allows to force set the build executor. Available values are 'job', 'dispatch', 'system' and ''. Default is empty.
  - **`g:cmake_vimspector_support`** enables generation and modification of [Vimspector](https://github.com/puremourning/vimspector) config file. Default is 0. **Attention! The support of Vimspector config is an experimental feature.**
  - **`g:cmake_variants`** enables predefined cmake build variants in the form of a dictionary, e.g. `{ 'Debug' : { 'cmake_build_type' : 'Debug', 'cmake_usr_args' : '-DCONAN_PATH=~/.conan' } }`
+ - **`g:cmake_ctest_args`** enables arguments for `ctest`, e.g. `[ '-j8', '--output-on-failure', '--verbose' ]`. Default is `[]`. If the user calls `:CTest -R "test-pattern"`, the `args` are inserted directly after `ctest`, before `-R` parameter.
```

Comment:
You have added `g:cmake_ctest_args` as an array, but other options (`g:make_arguments`, `g:cmake_usr_args` and etc.) are strings, I propose to use the unified approach for all options.